clang/lib/CodeGen/SwiftCallingConv.cpp

```diff
@@ -800,7 +800,9 @@ static ABIArgInfo classifyExpandedType(SwiftAggLowering &lowering,
   if (lowering.empty()) {
     return ABIArgInfo::getIgnore();
   } else if (lowering.shouldPassIndirectly(forReturn)) {
-    return ABIArgInfo::getIndirect(alignmentForIndirect, /*byval*/ false);
+    return ABIArgInfo::getIndirect(alignmentForIndirect,
+                                   /*AddrSpace*/ 0,
+                                   /*byval*/ false);
```

Comment:
Swift should use the general rules for the target.  Note that `forReturn` indicates whether we have an argument (false) or a return value (true).

This site is for types that are passed indirectly because they're too large; the site below is for types that are non-trivial.

Reply:
I did not intend to alter behaviour here, but merely to adopt the changed interface. There's no easy way to use she LangAS::Default -> Target mapping here in this helper function, hence the hardcoding to 0.

Reply:
I'm asking you to change behavior in a way that's consistent with the way you're changing behavior in general in your patch.